kolibri/plugins/learn/assets/src/views/ChannelCard.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+
+  <router-link
+    :to="link"
+    class="card"
+    :style="{ backgroundColor: $themeTokens.surface, color: $themeTokens.text }"
+  >
+    <div
+      class="card-heading"
+      :style="{ borderBottom: `1px solid ${$themeTokens.fineLine}` }"
+    >
+      <h3 class="title" dir="auto">
+        <TextTruncator
```

Comment:
I think `TextTruncator` should be used in the card-content for the tagline, not for the title